paper/figures/eady.jl

```diff
@@ -0,0 +1,130 @@
+using OceanBioME, Printf, Oceananigans, Oceananigans.Units
+
+using Oceananigans.Architectures: arch_array
+
+using Oceananigans.Solvers: FFTBasedPoissonSolver 
+
+using Random
+
+Random.seed!(11)
+
+h(k) = (k - 1) / Nz
+
+ζ₀(k) = 1 + (h(k) - 1) / refinement
+
+Σ(k) = (1 - exp(-stretching * h(k))) / (1 - exp(-stretching))
+
+z_faces(k) = Lz * (ζ₀(k) * Σ(k) - 1)
+
+Nz = 64
+Lz = 140  
+
+refinement = 1.8
+stretching = 3
+
+arch = GPU()
+
+grid = RectilinearGrid(arch; size = (512, 512, Nz), x = (0, 1kilometer), y = (0, 1kilometer), z = z_faces)
+
+Lx, Ly, Lz = grid.Lz, grid.Ly, grid.Lz
+
+coriolis = FPlane(f = 1e-4) # [s⁻¹]
+
+background_state_parameters = ( M = 1e-4,       # s⁻¹, geostrophic shear
+                                f = coriolis.f, # s⁻¹, Coriolis parameter
+                                N = 1e-4,       # s⁻¹, buoyancy frequency
+                                H = grid.Lz )
+
+# We assume a background buoyancy ``B`` with a constant stratification and also a constant lateral
+# gradient (in the zonal direction). The background velocity components ``U`` and ``V`` are prescribed
+# so that the thermal wind relationship is satisfied, that is, ``f \partial_z U = - \partial_y B`` and
+# ``f \partial_z V = \partial_x B``.
+B(x, y, z, t, p) = p.M ^ 2 * x + p.N ^ 2 * (z + p.H)
+V(x, y, z, t, p) = p.M ^ 2 / p.f * (z + p.H)
+
+V_field = BackgroundField(V, parameters = background_state_parameters)
+B_field = BackgroundField(B, parameters = background_state_parameters)
+
+νᵥ = κᵥ = 1e-4 # [m² s⁻¹]
+vertical_diffusivity = VerticalScalarDiffusivity(ν = νᵥ, κ = κᵥ)
+
+@info "Setting up kelp particles"
+
+n = 32
+
+x = arch_array(arch, [repeat([-200.0, -100.0, 0.0, 100.0, 200.0], 1, 5)...] .+ Lx / 2)
```

Comment:
```suggestion
x = arch_array(arch, [repeat([-200.0, -100.0, 0.0, 100.0, 200.0], 1, 5)...]  .+ Lx / 2)
```

Reply:
I'll update this since there was an issue here too